docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+# Hint: run `docker build --tag fingerprint-pro-server-api-dotnet:latest ./` to prepare image required for these service
+services:
+  # Generic service for running arbitrary dotnet commands
+  # Example usage: `docker compose run --rm dotnet dotnet list package`
```

Comment:
`dotnet` twice intentional?

Reply:
Yep, first `dotnet` is the name of service, second one is calling the `dotnet` CLI.